Library/Formula/node.rb

```diff
@@ -24,10 +24,7 @@ class Node < Formula
   depends_on "openssl" => :optional
 
   # Per upstream - "Need g++ 4.8 or clang++ 3.4".
-  # Clang should work *above* Snow Leopard but currently doesn't.
-  # https://github.com/nodejs/node/issues/4284
-  # Fix is landing in v5.3.0, so line below can revert to snow_leopard then.
-  fails_with :clang if MacOS.version <= :lion
+  fails_with :clang if MacOS.version <= :snow_leopard
```

Comment:
Heh, I'm planning to be offline for most of the night and I thought it may prove useful with the release in the process of being cut :wink: